webapp-django/crashstats/monitoring/tests/test_views.py

```diff
@@ -278,3 +280,42 @@ def fake_es_instance(**config):
         )
 
         eq_(es_instance.info.call_count, 2)
+
+    def test_assert_supersearch_counts(self):
+
+        searches = []
```

Comment:
You don't use the content of ``searches`` anywhere, might as well make it a counter?